### ISSUE TITLE
headless install doesn't seem to work with bookworm

### ISSUE THREAD

**Issue by Aaronsss** [Aaronsss/RH-Setup#6]
The automatic install doesn't seem to work on bookworm (Pi OS 12). You can install manually without any issues

Automatic install does still work on bullseye (PI OS 11)

**Comment by martinglass** [comment]
Not just me then! ;)

**Comment by Aaronsss** [comment]
yeah i need to have a bit more of a look into it. What I can see so far is bookworm is deleting the RH folder
